### DISCUSSION TITLE
Slider curve type manipulation by keystrokes and mouse hold in editor

### DISCUSSION THREAD

**Discussion by pethrol** [ppy/osu#25829] [Ideas]
I thought it would be awesome if you could change curve types by holding an anchor using mouse and then press 1, 2, 3... etc for each specified anchor type.

I prepared a little visual demo. Keystrokes are theoretical.

https://github.com/ppy/osu/assets/22896290/60760394-cf3b-4dff-903e-d3eeec215a4a

I think thats even better than pie menus proposed by minetoblend https://github.com/ppy/osu/discussions/24482

**Comment by Skubers** [comment]
Personally I really prefer what is in stable right now, so for me, just clicking a bunch of times on the anchor would rotate between the curve type choices. No menus required.

**Reply by pethrol** [reply]
Im personally just used to stable, its not the best what I consider optimal. Current state of this feature in lazer its just painful to use. 

Your way also works well plus its important to notice that the slider anchor grouping feature exists. I dont know if anybody would make much use of that combination.
